src/cmd/check.go

```diff
@@ -85,19 +139,18 @@ var deleteCheckCmd = &cobra.Command{
 }
 
 func init() {
+	checkCreateCmd.Flags().Bool("noninteractive", false, "don't use promptui - all linked fields must be defined in yaml")
 	createCmd.AddCommand(checkCreateCmd)
+	checkUpdateCmd.Flags().Bool("noninteractive", false, "don't use promptui - all linked fields must be defined in yaml")
```

Comment:
same documentation change as right above
```suggestion
	checkUpdateCmd.Flags().Bool("noninteractive", false, "turns off automated prompts for fields missing data in the spec")
```